chapters/annotations.tex

```diff
@@ -1320,11 +1320,13 @@ \subsubsection{Bitmap}\label{bitmap}
 
 \subsection{Variable Graphics and Schematic Animation}\label{variable-graphics-and-schematic-animation}
 
-Any value (coordinates, color, text, etc.) in graphical annotations can be dependent on class variables using \lstinline!DynamicSelect!.
-\lstinline!DynamicSelect! has the syntax of a function call with two arguments, where the first argument specifies the value of the editing state and the second argument the value of the non-editing state.
-The first argument must be a literal expression.
+Any value in graphical annotations can be dependent on class variables using \lstinline!DynamicSelect! except when restricted otherwise in their respective definitions, (for example with \lstinline!/* literal */ constant!).
+\lstinline!DynamicSelect! has the syntax of a function call with two arguments, where the first argument specifies the value of the static state and the second argument the value of the dynamic state.
+The first argument follows the rule given below.
```

Comment:
The first argument follows the same rules as when not using \lstinline!DynamicSelect!.

Reply:
This was supposed to be a "suggestion"...

```suggestion
The first argument follows the same rules as when not using \lstinline!DynamicSelect!.
```